linux/build.sh

```diff
@@ -15,6 +16,7 @@ cd "$THIS_SCRIPT_PATH"
 builder_describe \
   "Build Keyman for Linux." \
   ":engine=ibus-keyman       ibus-keyman" \
+  ":config=keyman-config     keyman-config" \
```

Comment:
Would keyman-config be considered the "app"? e.g.
```suggestion
  ":app=keyman-config     Keyman Configuration" \
```

Reply:
I think "app" is an awkward model for all the desktop versions of Keyman. Let's stick with "config" for keyman-config.